components/engagement/EngSelectContacts.vue

```diff
@@ -41,65 +44,106 @@
         </div>
       </div>
     </form>
-    <div v-if="isSelected1" class="show-contact1">
-      <show-contacts
-        :contact-name="contactName"
-        :department="department"
-        :contact-email="contactEmail"
-        :last-eng-title="lastEngTitle"
-        :engagement-date="engagementDate"
-        :num-participants="numParticipants"
-        :is-selected="isSelected1"
-      />
-      <button @click="isSelected1=false">
-        {{ $t ('engSelect.remove') }}
-      </button>
+    <div v-if="isSelected === true">
+      <div>
+        <display-contact
+          :name="contactName"
+          :org-name="orgName"
+          :email="contactEmail"
+          :type="engagementType"
+          :date="engagementDate"
+          :participants="participants"
+        />
+      </div>
+      <div>
+        <button @click="isSelected=false">
+          {{ $t ('engSelect.remove') }}
+        </button>
+      </div>
     </div>
+    <!--
     <div v-if="isSelected2" class="show-contact2">
-      <show-contacts
-        :contact-name="contactName"
-        :department="department"
-        :contact-email="contactEmail"
-        :last-eng-title="lastEngTitle"
-        :engagement-date="engagementDate"
-        :num-participants="numParticipants"
-        :is-selected="isSelected2"
-      />
+      display contact info here
       <button @click="isSelected2=false">
         {{ $t ('engSelect.remove') }}
       </button>
     </div>
+    -->
   </div>
 </template>
 
 <script>
-import AppSelect from '../app/AppSelect'
-import showContacts from './EngShowContacts'
+import displayContact from '../app/AppDisplayContact'
+import formSelect from './EngFormSelect'
 export default {
   components: {
-    AppSelect,
-    showContacts
+    formSelect,
+    displayContact
   },
   data() {
     return {
-      contactName: 'contact name',
-      department: 'Government of Canada',
-      contactEmail: 'email@email.com',
-      lastEngTitle: 'last engagement title',
-      engagementDate: '01/05/2020',
-      numParticipants: 0,
-      contacts: [
-        { key: 1, value: 'contact1', options: 'contact1' },
-        { key: 2, value: 'contact2', options: 'contact2' },
-        { key: 3, value: 'contact3', options: 'contact3' }
-      ],
+      contactName: '',
+      orgName: '',
+      title: '',
+      contactEmail: '',
+      phoneNum: '',
+      lastEngId: '',
+      engagementType: '',
+      engagementDate: new Date(),
+      participants: 0,
+      contacts: [],
+      engagements: [],
       moreContacts: false,
-      isSelected1: false,
-      isSelected2: false
+      isSelected: false
+    }
+  },
+  // eslint-disable-next-line space-before-function-paren
+  async created() {
+    try {
+      const res = await this.$axios.get('/api/contact/contacts')
```

Comment:
use the Vuex store call for this call

Reply:
I'll do this on a new branch, and do a pr after

Reply:
the boys are gonna set up a separate PR for that